pebble/concurrent/process.py

```diff
@@ -47,24 +47,29 @@ def process(*args, **kwargs):
     """
     timeout = kwargs.get('timeout')
     name = kwargs.get('name')
+    daemon = kwargs.get('daemon')
```

Comment:
Rather than forcing a `True` value in the wrapper down below, you can default it here and pass the variable itself.

```
daemon = kwargs.get('daemon', True)

...
    return _process_wrapper(args[0], timeout, name, daemon)
```

Reply:
You are completely right.